### ISSUE TITLE
Unable to install doublecmd-snapshot

### ISSUE THREAD

**Issue by clin1234** [tleepa/scoop-bucket#1]
Got the following output:
```
Installing 'doublecmd-snapshot' (1.2.0.r11352) [64bit] from 'tleepa_scoop-bucket' bucket
Loading doublecmd-1.2.0.r11352.x86_64-win64.7z from cache.
Checking hash of doublecmd-1.2.0.r11352.x86_64-win64.7z ... ok.
Extracting doublecmd-1.2.0.r11352.x86_64-win64.7z ... done.
Linking ~\scoop\apps\doublecmd-snapshot\current => ~\scoop\apps\doublecmd-snapshot\1.2.0.r11352
Creating shim for 'doublecmd'.
Get-Command : The term 'doublecmd.exe' is not recognized as the name of a
cmdlet, function, script file, or operable program. Check the spelling of
the name, or if a path was included, verify that the path is correct and
try again.
At C:\Users\ךינשגכהד\scoop\apps\scoop\current\lib\install.ps1:757 char:21
+             $bin = (Get-Command $target).Source
+                     ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (doublecmd.exe:String) [Get-
   Command], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException,Microsoft.PowerShel
   l.Commands.GetCommandCommand

Can't shim 'doublecmd.exe': File doesn't exist.
```

**Comment by tleepa** [comment]
It looks like the problem with the upstream package - it does not include the executable:
```
❯ 7z l .\Downloads\doublecmd-1.2.0.r11352.x86_64-win64.7z doublecmd.*
[...]
   Date      Time    Attr         Size   Compressed  Name
------------------- ----- ------------ ------------  ------------------------
2024-06-01 19:30:07 ....A            2               doublecmd.help
2024-06-01 19:30:07 ....A          357               doublecmd.visualelementsmanifest.xml
------------------- ----- ------------ ------------  ------------------------
2024-06-01 19:30:07                359      4905526  2 files
```

**Comment by tleepa** [comment]
Closing, as `doublecmd.exe` is included in latest version (1.2.0.r11356 as of now).
